dist/stat_data.py

```diff
@@ -1177,7 +1177,9 @@ def __init__(self, name, desc, flags=''):
 ##########################################
 session_stats = [
     SessionStat('bytes_read', 'bytes read into cache'),
+    SessionStat('bytes_read_app_count', 'io operation count for bytes read into cache'),
     SessionStat('bytes_write', 'bytes written from cache'),
+    SessionStat('bytes_write_app_count', 'io operation count for bytes written from cache'),
```

Comment:
I think it would be clearer to use parallel names and description for these new statistics.

Maybe `blocks_read_count` and `blocks_write_count`, described as `blocks read/written into/from cache`.

To me saying "io operations" means device-level read/write requests. That's not what these statistics are measuring.

Having said that, I'm not sure how useful these new stats actually are. If we could count actual IO requests (i.e., at the device level) that would definitely be useful. But the block requests measured here might be satisfied from the block cache (not used by MongoDB), the OS page cache, or the device. So it seems that the primary information we would get from them is the average size of the blocks being read/written into the cache. I'm not sure if knowing this for a specific request is more useful than the aggregate numbers already available in the block manager statistics.

Reply:
Hi @y123456yz , it looks like we now have duplicate stats for write_io and read_io.

Reply:
@ruby-oujo 
mongodb's log:  [ "storage":{"data":{"bytesRead":16674970473,"timeReadingMicros":52487124}} ].

WT have "WT_STAT_CONN_INCR(session, write_io)",  This is a conn level log statistics.
but We also need a session level log statistics, so I add "WT_STAT_SESSION_INCR(session, write_io)".

After this PR, I will add a new PR in mongoserver to Adapt this feature. the new mongodb's log will be following:
 [ "storage":{"data":{"bytesRead":16674970473, "write io":xxxxx, "timeReadingMicros":52487124}} ].